sm/plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -15,6 +15,7 @@ pub enum PluginOp {
     /// Install a module
     Install {
         module: String,
+        #[structopt(short = "v", long = "--module-version")]
```

Comment:
The exact same thing must be done for the `remove` sub-command.

Reply:
Added

Reply:
For me it looks good. I hope the UX is correct now (-v and --module-version).